src/JobPanel/backend/connection.py

```diff
@@ -443,7 +445,8 @@ def get_delegator(self, local_service):
                     connected = True
                     break
             if connected:
-                self._delegator_proxy = ServiceProxy(local_service._repeater, {}, self)
+                from .service_proxy import ServiceProxy
```

Comment:
Teoreticky by to šlo, ale má to taky svoje mouchy.
- Když jsou moduly zacykleny, není potom možný na úrovni modulu použít něco z druhého modulu, třeba nejde odvozovat třídu, která je definovaná v jiném modulu.
- Není možné použít "from backend import yy", museli bychom používat "import backend.xx" a pak se všude odkazovat s plnou cestou backend.xx.yy.